rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/TaskControllerIntTest.java

```diff
@@ -565,6 +565,44 @@ void should_NotGetEmptyAttachmentList_When_GettingTaskWithAttachment() {
     assertThat(repModel.getAttachments()).isNotNull();
   }
 
+  @Test
+  void should_ReturnReceivedDate_When_GettingTask() {
+    String url =
+        restHelper.toUrl(RestEndpoints.URL_TASKS_ID, "TKI:000000000000000000000000000000000024");
+    HttpEntity<Object> auth = new HttpEntity<>(RestHelper.generateHeadersForUser("admin"));
+
+    ResponseEntity<TaskRepresentationModel> response =
+        TEMPLATE.exchange(url, HttpMethod.GET, auth, TASK_MODEL_TYPE);
+
+    TaskRepresentationModel repModel = response.getBody();
+    assertThat(repModel).isNotNull();
+    assertThat(repModel).extracting(TaskSummaryRepresentationModel::getReceived).isNotNull();
```

Comment:
combine assertions :)

Reply:
actually combine, or just drop the first?